### PR TITLE
[Bug] profile photo of brand Fixed

### LINKED ISSUE #295: [Bug] profile photo of brand
Fixing profile photo of brand to allow they loved avatars.

### PR DESCRIPTION
**Pull request by Hemu21**
## Issue that this pull request solves

[Issue Link](https://github.com/Kajol-Kumari/InfluMart/issues/295) resolve #295 

Closes: #295 

## Proposed changes

Allow brands to add they loved avatars.

### Brief description of what is fixed or changed

I update the backend and frontend to accept the user lover avatars and integrated both frontend and backend to make this funtional.

## Screenshot or Video

https://github.com/user-attachments/assets/c6fe371f-30bf-4495-9751-00e0120bf551

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by Hemu21** [comment]
@Kajol-Kumari  @RohitGupta1237  can you please check

File: frontend/screens/BrandAccountSignupDataPreview.js [hunk 1]
```diff
@@ -64,32 +106,107 @@ const BrandAccountSignupDataPreview = ({ route, navigation }) => {
...
+            file: `avatar${selectedAvatarIndex + 1}`,
+          });
+          setSelectedImage(base64);
+          console.log(selectedAvatarIndex, `avatar${selectedAvatarIndex + 1}`);
+        }
+      );
+    }
+  }, [selectedAvatarIndex]);
+
+  const avatarImages = [
```
Comment (Kajol-Kumari) [inline frontend/screens/BrandAccountSignupDataPreview.js:160]: can these be kept at common place as its getting used for influencers as well right?

**Review by Kajol-Kumari — COMMENTED** [review]
@Hemu21 code-wise looks good. can you attach recording as well showing the functionality

File: frontend/screens/BrandAccountSignupDataPreview.js (hunk 1 above)
Comment (Hemu21) [reply frontend/screens/BrandAccountSignupDataPreview.js:160]: Ok I will update.

**Comment by Hemu21** [comment]
@Kajol-Kumari  @RohitGupta1237  updated can you please check.

**Pull request merged**